


export const addProduct = async (req, res) => {

    try {

    } catch (error) {

        console.log(error)
        return res.json({ success: false, error })
    }

}




export const getProduct = async (req, res) => {

    try {


    } catch (error) {

        console.log(error)
        return res.json({ success: false, error })
    }

}



export const updateProduct = async (req, res) => {

    try {


    } catch (error) {

        console.log(error)
        return res.json({ success: false, error })
    }

}


export const deleteProduct = async (req, res) => {

    try {


    } catch (error) {

        console.log(error)
        return res.json({ success: false, error })
    }

}